README.md

```diff
@@ -41,84 +50,27 @@ git clone https://github.com/neuralet/smart-social-distancing.git
 ./download_sample_video.sh
 ```
 
-#### Web App
-(This step is optional if you are not going to build any docker image)
-
-The frontend consists of 2 Dockerfiles:
-* `frontend.Dockerfile`: Builds the React app.
-* `web-gui.Dockerfile`: Builds a FastAPI backend which serves the React app built in the previous Dockerfile.
-
-If the `frontend` directory on your branch is not identical to the upstream `master` branch, you MUST build the frontend image with
-tag "`neuralet/smart-social-distancing:latest-frontend`" BEFORE BUILDING THE MAIN FRONTEND IMAGE.
-Otherwise, skip this step, as we have already built the frontend for `master` branch on Dockerhub.
-
-* To build the frontend run:
-
-```bash
-docker build -f frontend.Dockerfile -t "neuralet/smart-social-distancing:latest-frontend" .
-```
-
-* To run the frontend, run:
-
-```bash
-docker build -f web-gui.Dockerfile -t "neuralet/smart-social-distancing:latest-web-gui" .
-docker run -it -p HOST_PORT:8000 --rm neuralet/smart-social-distancing:latest-web-gui
-```
-
-> Important: There is a `config-frontend.ini` file which tells the frontend where to find the processor container.
-> You must set the "Processor" section of the config file with the correct IP and port of the processor.
-
----
-***NOTE***
-
-Building the frontend is resource intensive. If you are planning to host everything on an edge device, we suggest building the docker image on your PC/laptop first and then copy it to the edge device. However, you can always start the frontend container on a PC/laptop and the processor container on the edge device.
-
----
-
-* To run the frontend on an edge device (Only possible on jetson for now):
-
-```bash
-# Run this commands on your PC/laptop:
-docker build -f frontend.Dockerfile -t "neuralet/smart-social-distancing:latest-frontend" .
-docker save -o "frontend_base_image.tar" neuralet/smart-social-distancing:latest-frontend
-```
-
-* Then, move the file `frontend_base_image.tar` that was just built on your PC/laptop to your jetson platform and load it:
-```bash
-# Copy "frontend_image.tar" to your edge device and run this command on your device:
-docker load -i "frontend_base_image.tar"
-rm frontend_base_image.tar
-```
-
-* Then build the web-gui image for jetson:
-```bash
-docker build -f jetson-web-gui.Dockerfile -t "neuralet/smart-social-distancing:latest-web-gui-jetson" .
+### Usage
+The smart social distancing app consists of two components: the `frontend` and the `processor`. 
 
-# And run it:
-docker run -it -p HOST_PORT:8000 --rm neuralet/smart-social-distancing:latest-web-gui-jetson
-```
+#### Frontend
+The frontend is a public [webapp](http://beta.lanthorn.ai) provided by [lanthorn](https://www.lanthorn.ai/) where you can signup for free. This web app allows you to configure some aspects of the processor (such as notifications and calibration) using a friendly UI. Moreover, it provides a dashboard that helps you to analyze the data that your cameras are processing. 
```

Comment:
We normally prefer to have each sentence on a new line in markdown files so that PR comments are easier to process.

```suggestion
The frontend is a public [web app](http://beta.lanthorn.ai) provided by [lanthorn](https://www.lanthorn.ai/) where you can signup for free. 
This web app allows you to configure some aspects of the processor (such as notifications and camera calibration) using a friendly UI. 
Moreover, it provides a dashboard that helps you to analyze the data that your cameras are processing. 
```